RFC/src/RFC-0201_TariScript.md

```diff
@@ -524,6 +524,10 @@ $$
 For Bob's part, when he discovers one-sided payments to himself, he should spend them to new outputs using a traditional
 transaction to thwart any potential horizon attacks in the future.
 
+One other problem with one sided communication is the communication of the value of the transaction being sent, and the wallet recovery of one sided payments. We can solve both of these by using (\\( k_{Ob} \\).
+Because this is a secret key known by Alice and Bob and no one else. We hash this key as a new key, (\\( k_{rewind} =  Hash(k_{Ob}) \\). We know have a rewinding secret key for use in bulletproof rewinding. And now
```

Comment:
We ~~know~~ _now_ have ...

Reply:
```suggestion
Because this is a secret key known by Alice and Bob and no one else. We hash this key as a new key, (\\( k_{rewind} =  Hash(k_b) \\). We know have a rewinding secret key for use in bulletproof rewinding. And now
```